deployment/build_and_release/modules/release/main.tf

```diff
@@ -270,7 +270,7 @@ resource "google_cloudbuild_trigger" "applet_build" {
       name = "bash"
       script = (
         var.cloudbuild_trigger_tag != "" ?
-        "$TAG_NAME > /workspace/git_tag && cat /workspace/git_tag" :
+        "echo $TAG_NAME > /workspace/git_tag && cat /workspace/git_tag" :
```

Comment:
Why this change? I can't see how the other changes would lead to this being needed now if it was OK before?

Reply:
This fixing a bug which we haven't run into because we haven't run prod yet. (This is one of the few places where CI and prod deviate.)